fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -165,7 +185,10 @@ def isNotForbbiden(value) -> bool:
     Returns:
         bool: Returns True as long as the given exception value is not InvalidStatusCode with 401 or 403
     """
-    return not (isinstance(value, InvalidStatusCode) and (value.status_code == 401 or value.status_code == 403))
+    return not (
+        isinstance(value, InvalidStatusCode)
+        and (value.status_code == 401 or value.status_code == 403)
```

Comment:
would use value.status_code in (401,403)

Reply:
That's just reformatting